packages/gatsby-remark-images-contentful/src/index.js

```diff
@@ -0,0 +1,186 @@
+const select = require(`unist-util-select`)
+const sharp = require(`sharp`)
+const axios = require(`axios`)
+const _ = require(`lodash`)
+const Promise = require(`bluebird`)
+const cheerio = require(`cheerio`)
+const { buildResponsiveSizes } = require(`./utils`)
+
+// If the image is hosted on contentful
+// 1. Find the image file
+// 2. Find the image's size
+// 3. Filter out any responsive image sizes that are greater than the image's width
+// 4. Create the responsive images.
+// 5. Set the html w/ aspect ratio helper.
+
+module.exports = async (
+  { files, markdownNode, markdownAST, pathPrefix, getNode, reporter },
+  pluginOptions
+) => {
+  const defaults = {
+    maxWidth: 650,
+    wrapperStyle: ``,
+    backgroundColor: `white`,
+    linkImagesToOriginal: true,
+    showCaptions: false,
+    pathPrefix,
```

Comment:
linkImagesToOriginal, showCaptions and pathPrefix aren't really used anywhere? Or I'm missing something?

Reply:
for some reason my changes were not pushed to the branch 🙈

Reply:
Done